src/channel.js

```diff
@@ -257,4 +260,10 @@ export default class Channel {
     }
     this._dispatchInnerRemoveEvents.apply(this, args);
   }
+
+  _throwIfDispatching(message) {
```

Comment:
Maybe add [throw-utils](https://www.npmjs.com/package/throw-utils) and use directly instead of this method.
```js
throwIf(this. _isDispatching, 'Removing listeners in dispatching loop');
```